panoptes_cli/scripts/panoptes.py

```diff
@@ -41,10 +43,44 @@ def cli(ctx, endpoint, admin):
         ctx.config['endpoint'] = endpoint
 
     if ctx.invoked_subcommand != 'configure':
+        try:
+            password = keyring.get_password('panoptes', ctx.config['username'])
+        except RuntimeError:
+            password = None
+
+        if 'password' in ctx.config:
+            if not password:
+                try:
+                    password = ctx.config['password']
+                    keyring.set_password(
+                        'panoptes',
+                        ctx.config['username'],
+                        password,
+                    )
+                    retrieved_password = keyring.get_password(
+                        'panoptes', 
+                        ctx.config['username'],
+                    )
```

Comment:
To do: Compare `password` and `retrieved_password` here to verify keyring is working.